evm/bridge-token-factory/contracts/BridgeTokenFactory.sol

```diff
@@ -151,7 +66,7 @@ contract BridgeTokenFactory is
             revert InvalidSignature();
         }
 
-        require(!_isBridgeToken[_nearToEthToken[metadata.token]], "ERR_TOKEN_EXIST");
+        require(!isBridgeToken[nearToEthToken[metadata.token]], "ERR_TOKEN_EXIST");
```

Comment:
Let's also replace strings with custom errors. Btw, in 0.8.26 Solidity allows to use custom errors in require statements 
https://soliditylang.org/blog/2024/05/21/solidity-0.8.26-release-announcement/

Reply:
I'm getting this error 
```
UnimplementedFeatureError: Require with a custom error is only available using the via-ir pipeline.
Error HH600: Compilation failed
```

Reply:
It can be turned on this way in the config file https://ethereum.stackexchange.com/a/142206

In general, do we want to switch to IR pipeline? I think that's fine, it's the newer approach https://soliditylang.org/blog/2024/07/12/a-closer-look-at-via-ir/

Reply:
Ok, let's add and test it in a follow-up PR